StockTicker/app/src/main/java/com/example/stockticker/MainActivityViewModel.kt

```diff
@@ -9,16 +10,15 @@ import com.example.stockticker.view.MainActivityUiModel
 import com.example.stockticker.view.StockUiModel
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.FlowPreview
-import kotlinx.coroutines.InternalCoroutinesApi
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.launch
 import java.math.BigDecimal
 import java.math.RoundingMode
-import kotlin.math.round
+import javax.inject.Inject
 
 @FlowPreview
 @ExperimentalCoroutinesApi
-class MainActivityViewModel(
+class MainActivityViewModel @Inject constructor(
```

Comment:
This can be `@ViewModelInject` if this view model is only to be used in a fragment.